app/views/users/_table.html.erb

```diff
@@ -21,9 +21,6 @@
               <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">
                 Reviews
               </th>
-              <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">
```

Comment:
Waarom moet deze column weg? Ik vind het nog steeds fijn op het moment om te weten wie 2FA uitzet.

Reply:
Dacht dat we hadden afgesproken dat hij weg ging zodra iedereen het had geactiveerd? We kunnen 2FA gewoon verplicht maken, dan is de kolom zeker niet meer nodig.

Verder is het een beetje arbitrair wat wel en wat niet in die kolom gaat natuurlijk

Reply:
Ik had het ook in mijn hoofd dat we de column zouden houden tot 2FA uiteindelijk verplicht is vandaag 😅

Reply:
Dan gewoon verplicht maken toch?